src/VS1053.h

```diff
@@ -113,7 +115,8 @@ class VS1053 {
     VS1053(uint8_t _cs_pin, uint8_t _dcs_pin, uint8_t _dreq_pin);
 
     // Begin operation.  Sets pins correctly, and prepares SPI bus.
-    void begin();
+    // The patch parameter can be used to prevent the latest firmware patch from loading.
+    void begin(const bool autopatch = true);
```

Comment:
I'm wondering how about this one. Even the default parameter feature would work here to avoid breaking a backward compatibility.

This is a little bit magic.
If we know that not all patches are compatible with the latest firmware, this will cause a lot of troubles to users who will only update a library version. They will get a surprise that something will start failing for no reason.

By default it will load this firmware. This might be not expected behavior for all use cases. 

How about creating a method `loadDefaultVs1053Patches()` which would call the `loadUserCode(patches)`?

Then a default proposed library use case example in README.md would be like:
```c
vs1053->begin();
vs1053->loadDefaultVs1053Patches();
vs1053->switchToMp3Mode();
```
But we would not force everyone to use patches in a hidden way. However, newcomers would copy this code snippet and use it by default anyway. 
If one day they want to change their mind, the above code is clear enough (without studying documentation) to spot out what to do. That would see that maybe here is an interesting thing. This would lead them to jump directly to interesting fragment of documentation and learn about a possibility for calling `loadUserCode(aPatchOfTheirChoice)` instead.

What do you think?

PS: Also, an argument like `true` says not a lot when seen in a code. In general boolean flags are not about good programming pattern. If we really need to do so, then enumeration would be better here. There is even a code smell called "Boolean blindness". It's always safer and cleaner to encode as much information as possible into types.